TikzFix/Model/Tool/ITool.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace TikzFix.Model.Tool
+﻿using System.Windows.Shapes;
+
+namespace TikzFix.Model.Tool
 {
     interface ITool
     {
+        // todo, change x/yscale to 1 and scale shape to fill generated drawing
+        public const string TIKZ_MAIN = "\\begin{{tikzpicture}}[xscale=0.1, yscale=-0.1]\n{0}\n\\end{{tikzpicture}}"; // y scale has to be negative, tikz is mirrored vertically
+
         DrawingShape GetShape(CanvasEventArgs canvasEventArgs);
+
```

Comment:
It would be better to break this interface into two right now. Tools cant generate JSON or TikZ code, neither generators cant create new shapes. Breaking interfaces apart gives you more granular control over components in the application.
```suggestion
interface ITool
{
 DrawingShape GetShape(CanvasEventArgs canvasEventArgs);
}

interface IGenerator //or replace with better fitting name
{
public const string TIKZ_MAIN = "\\begin{{tikzpicture}}[xscale=0.1, yscale=-0.1]\n{0}\n\\end{{tikzpicture}}"; // y scale has to be negative, tikz is mirrored vertically

LocalShapeData ConvertToShapeData(Shape shape);

string GenerateTikzShape(Shape shape);
}
```

You can always inherit from multiple interfaces :)